components/Settings.js

```diff
@@ -101,7 +101,7 @@ const Settings = (props) => {
         { !regionKey && enableInvitations && (
           <SettingsItem term="settings/accept_invites" onPress={acceptInvite} />
         )}
-        <View style={[SettingsItem.styles.container, { minHeight: 3 * fonts.large }]}>
+        <View style={[SettingsItem.styles.container, { maxHeight: 1.5 * fonts.large }]}>
```

Comment:
@smaclell Is there a reason the language picker had extra height compared to the other settings items?

Reply:
I see. On Android, it doesn't show you the edges of multiple items it's more of like a dropdown which is why I couldn't tell. I suppose moving it to the top would work as well